images/base/files/usr/local/bin/mount-product-files

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is a createContainer hook [1] that replicates the functionality from entrypoint script to mount product_name and product_uuid but from a product_name and product_uuid copied into the contianer rootfs to prevent all the containers from bind mounting the same file. Sharing the same bind mount between all the containers increases the latency accessing the container, preventing it from accessing in some cases.
+#
+# [1] https://github.com/opencontainers/runtime-spec/blob/master/config.md#createcontainer-hooks
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# The bundle represents the dir path to container filesystem, containre spec [1] is
+# passed to the hook's stdin
+#
+# [1] {"ociVersion":"1.0.2-dev","id":"2fdcb746d012f506eb6ddb69ee5f152f424865683b89bb1cda71515de6e7d693","status":"creating","pid":1,"bundle":"/run/containerd/io.containerd.runtime.v2.task/k8s.io/2fdcb746d012f506eb6ddb69ee5f152f424865683b89bb1cda71515de6e7d693","annotations":{"io.kubernetes.cri.container-name":"nginx","io.kubernetes.cri.container-type":"container","io.kubernetes.cri.image-name":"docker.io/library/nginx:latest","io.kubernetes.cri.sandbox-id":"579f074945c9dd65b0eb4defbcb39af09b7a50d6e53446c61e701f0e53987a36","io.kubernetes.cri.sandbox-name":"nginx-598b589c46-nzphj","io.kubernetes.cri.sandbox-namespace":"default"}}
+bundle=$(sed -r 's/.*,"bundle":"([/0-9a-z\.]*)",.*/\1/')
+
+cp /kind/product_* "$bundle/rootfs/"
```

Comment:
is there a reason we need to do a copy + bind mount? why not copy to the desired locations? or bind mount from the original locations?